docs/podman-rm.1.md

```diff
@@ -13,7 +13,8 @@ podman\-rm - Remove one or more containers
 
 **--force, f**
 
-Force the removal of a running container
+Force the removal of a running container. In the case that the container is paused, it will be sent
```

Comment:
Change to "Force the removal of running and paused containers

Reply:
fixed